### ISSUE TITLE
[9.4.5] The characters "\" are removed from description.

### ISSUE THREAD

**Issue by robsonjorgefernandez** [glpi-project/glpi#6899]
**Describe the bug**

When a technician click on `Associate Myself` all characters `\` are removed from description.

Ex:
     From: `t:\direcotry01\direcotry02\file01.txt`
     To: `t: direcotry01direcotry02file01.txt`

**To reproduce**

1. Open a ticket as a requester and add some text with `\`
2. As a technician associate the ticket with button `Associate Myself`
3. All characters  `\` are removed from description.

**Screenshots**

A ticket has been opened
![open01](https://user-images.githubusercontent.com/16673533/73692488-2b6ee680-46b3-11ea-89d5-4c4e4d49e8e7.png)

No change
![open02](https://user-images.githubusercontent.com/16673533/73692489-2c077d00-46b3-11ea-863e-9774e7173d7c.png)

Now associte the ticket with button `Associate Myself` and the characters `\` are removed
![associate01](https://user-images.githubusercontent.com/16673533/73692485-2b6ee680-46b3-11ea-899d-a93c62a78d41.png)

The characters `\` are removed
![associate02](https://user-images.githubusercontent.com/16673533/73692487-2b6ee680-46b3-11ea-95ef-4b56c0d17f93.png)

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

<details><summary>My system:</summary>
<p>
<pre>
{
    "glpi": {
        "uuid": "MRaizpbTXfhzZ2L5TTF3oGh69uCqutZNvF2NY6tR",
        "version": "9.4.5",
        "plugins": [
            {
                "key": "formcreator",
                "version": "2.9.1"
            },
            {
                "key": "fusioninventory",
                "version": "9.4+2.4"
            }
        ],
        "default_language": "pt_BR",
        "install_mode": "TARBALL",
        "usage": {
            "avg_entities": "0-500",
            "avg_computers": "500-1000",
            "avg_networkequipments": "0-500",
            "avg_tickets": "10000-50000",
            "avg_problems": "0-500",
            "avg_changes": "0-500",
            "avg_projects": "0-500",
            "avg_users": "500-1000",
            "avg_groups": "0-500",
            "ldap_enabled": true,
            "mailcollector_enabled": false,
            "notifications_modes": [],
            "notifications": [
                "mailing",
                "ajax"
            ]
        }
    },
    "system": {
        "db": {
            "engine": "MariaDB Server",
            "version": "10.3.21-MariaDB-log",
            "size": "8403.6",
            "log_size": "",
            "sql_mode": ""
        },
        "web_server": {
            "engine": "Apache",
            "version": "2.4.34"
        },
        "php": {
            "version": "7.1.30",
            "modules": [
                "Core",
                "date",
                "libxml",
                "openssl",
                "pcre",
                "zlib",
                "filter",
                "hash",
                "Reflection",
                "SPL",
                "session",
                "standard",
                "apache2handler",
                "bz2",
                "calendar",
                "ctype",
                "curl",
                "dom",
                "mbstring",
                "fileinfo",
                "ftp",
                "gd",
                "gettext",
                "iconv",
                "intl",
                "json",
                "ldap",
                "exif",
                "mysqlnd",
                "PDO",
                "Phar",
                "posix",
                "shmop",
                "SimpleXML",
                "soap",
                "sockets",
                "sqlite3",
                "sysvmsg",
                "sysvsem",
                "sysvshm",
                "tokenizer",
                "xml",
                "xmlwriter",
                "xsl",
                "zip",
                "mysqli",
                "pdo_mysql",
                "pdo_sqlite",
                "wddx",
                "xmlreader",
                "xmlrpc",
                "apcu",
                "Zend OPcache"
            ],
            "setup": {
                "max_execution_time": "30",
                "memory_limit": "256M",
                "post_max_size": "20M",
                "safe_mode": false,
                "session": "files",
                "upload_max_filesize": "5M"
            }
        },
        "os": {
            "family": "Linux",
            "distribution": "Red Hat Enterprise Linux Server release 7.7 (Maipo)",
            "version": "3.10.0-1062.4.3.el7.x86_64"
        }
    }
}
</pre>

**Comment by trasher** [comment]
Maybe try:
```diff

diff --git a/front/ticket.form.php b/front/ticket.form.php
index eb6e75d6ca..ddfc9bba92 100644
--- a/front/ticket.form.php
+++ b/front/ticket.form.php
@@ -155,7 +155,7 @@
 
 } else if (isset($_POST['addme_observer'])) {
    $track->check($_POST['tickets_id'], READ);
-   $input = array_merge($track->fields, [
+   $input = array_merge(Toolbox::addslashes_deep($track->fields), [
       'id' => $_POST['tickets_id'],
       '_itil_observer' => [
          '_type' => "user",
@@ -171,7 +171,7 @@
 
 } else if (isset($_POST['addme_assign'])) {
    $track->check($_POST['tickets_id'], READ);
-   $input = array_merge($track->fields, [
+   $input = array_merge(Toolbox::addslashes_deep($track->fields), [
       'id' => $_POST['tickets_id'],
       '_itil_assign' => [
          '_type' => "user",
```

**Comment by robsonjorgefernandez** [comment]
It's works!
@trasher, thank you!

**Comment by trasher** [comment]
Thank you for the quick feedback. This fix will be part of 9.4.6 release, we already have it ready on our side.

**Comment by trasher** [comment]
https://github.com/glpi-project/glpi/commit/ebca9b1a7e6a347e30273f281e3518f34e42c8c4
